runtimevar/runtimevar.go

```diff
@@ -482,14 +482,20 @@ func bytesDecode(b []byte, obj interface{}) error {
 // DecoderByName returns a *Decoder based on decoderName.
 // It is intended to be used by VariableURLOpeners in driver packages.
 // Supported values include:
-//   - (empty string), "bytes": Returns the default, BytesDecoder;
-//       Snapshot.Valuewill be of type []byte.
+//   - empty string: Returns the URLEncoder default, or BytesDecoder
```

Comment:
What is a `URLEncoder`? How to set the default?

Reply:
Sorry, should have been `URLOpener.Decoder`. I clarified the comment.